### PR TITLE
Added /handwired/ziyoulang_k3_mod

### PR DESCRIPTION
**Pull request by coomstoolbox**
## Description

Initial PR for adding ziyoulang k3 keyboard modify project.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by dunk2k — APPROVED** [review]

File: keyboards/handwired/ziyoulang_k3_mod/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                            _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                            _______, _______, _______, _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                            _______, _______, _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                            _______, _______, _______, _______,
+                                                                                                                                               _______,
+        _______,          _______, _______,                   _______,                            _______, MO(1),            _______,                            _______,          _______,
```
Comment (waffle87) [inline keyboards/handwired/ziyoulang_k3_mod/keymaps/default/keymap.c:42]: This won't have worked.
```suggestion
        _______,          _______, _______,                   _______,                            _______, _______,            _______,                            _______,          _______,
```

File: keyboards/handwired/ziyoulang_k3_mod/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,                            KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_CAPS,          KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,                             KC_P4,   KC_P5,   KC_P6,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,                            KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+                                                                                                                                               KC_UP,
+        KC_LCTL,          KC_LGUI, KC_LALT,                   KC_SPC,                             KC_RALT, MO(1),            KC_RCTL,                            KC_P0,            KC_PDOT,
+                                                                                                                                      KC_LEFT, KC_DOWN, KC_RGHT
```
Comment (waffle87) [inline keyboards/handwired/ziyoulang_k3_mod/keymaps/via/keymap.c:54]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by coomstoolbox** [comment]
@waffle87 
Corrected the corrections. Please let me know if there is anything else I should fix.

File: keyboards/handwired/ziyoulang_k3_mod/rules.mk [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```
Comment (fauxpark) [inline keyboards/handwired/ziyoulang_k3_mod/rules.mk:2]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Pull request closed without merging**